src/config/network/iwd.md

```diff
@@ -13,59 +13,62 @@ Install the `iwd` package and enable the `dbus` and `iwd` services.
-[iwd]# station <interface> disconnect                    # Disconnect from a network
-[iwd]# known-networks list                               # List known networks
-[iwd]# known-networks forget <network-name>              # Forget a known network
-```
-
-> Note: `iwctl` also supports passing commands as arguments.
-> 
-> ```
-> $ iwctl station <interface> get-networks
-> ```
-
 ## Configuration
 
-Consult the [upstream
+Consult the relevant manual pages and the [upstream
 documentation](https://iwd.wiki.kernel.org/networkconfigurationsettings) for
 options and examples available to configure the daemon and networks as described
 below.
 
 ### Daemon configuration
 
 The main configuration file is located in `/etc/iwd/main.conf`. If it does not
-exist, you may create it.
+exist, you may create it. It is documented in `iwd.config(5)`.
 
 ### Network configuration
 
-IWD stores information on known networks, and reads information on
-pre-provisioned networks from network configuration files located in
-`/var/lib/iwd`; IWD monitors the directory for changes. Network configuration
-filenames consist of the encoding of the SSID followed by `.open`, `.psk`, or
-`.8021x` as determined by the security type.
+Network configuration is documented in `iwd.network(5)`, including examples. IWD
+stores information on known networks, and reads information on pre-provisioned
+networks from network configuration files located in `/var/lib/iwd`; IWD
+monitors the directory for changes. Network configuration filenames consist of
+the encoding of the SSID followed by `.open`, `.psk`, or `.8021x` as determined
+by the security type.
 
 A simple example of a WPA2/PSK secured network configuration at
-`/var/lib/iwd/<ssid>.psk` contains the pre-shared key, and optionally the
-plaintext password.
+`/var/lib/iwd/<ssid>.psk` contains the plaintext password.
 
 ```
 [Security]
-PreSharedKey=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 Passphrase=<password>
 ```
+
+## Troubleshooting
+
+IWD can attach to your wireless interface (e.g. `wlan0`) before `udev` has the
+chance to rename it according to the consistent naming rules. If the device is
+already in use, `udev`'s renaming will fail, and there will be a `udevd` error,
+which might be printed to your screen, similar to this one:
+
+```
+[   39.441723] udevd[1100]: Error changing net interface name wlan0 to wlp59s0: Device or resource busy
+[   39.442472] udevd[1100]: could not rename interface '3' from 'wlan0' to 'wlp59s0': Device or resource busy
+```
+
+To solve this error, you can either disable consistent naming, by adding
+`net.ifnames=0` to your kernel [cmdline](../kernel.md#cmdline) or creating a
+symbolic link from `dev/null` to `/etc/udev/rules.d/80-net-name-slot.rules`,
+therefore masking the renaming rule; or configure the `iwd` service so it
+ignores interfaces with inconsistent names, which can be done by adding the
+following lines to `/etc/sv/iwd/conf`:
```

Comment:
This paragraph is confusing to me. Are there two possibilities (separated by the ';'), and then two possibilities within the first possibility?

Reply:
This one's been fixed by ahesford!